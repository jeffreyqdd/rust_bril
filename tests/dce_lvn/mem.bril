@main {
  size: int = const 1;
  v0: float = const 3.14;
  float_ptr: ptr<float> = alloc size; 
  store float_ptr v0;
  v1: float = load float_ptr;
  v2: float = load float_ptr;
  v3: float = fadd v1 v2;
  free float_ptr;
  print v3;
}