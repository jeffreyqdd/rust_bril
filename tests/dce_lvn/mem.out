6.28000000000000025
